src/discord-cluster-manager/run_eval.py

```diff
@@ -204,7 +204,7 @@ def compile_cuda_script(  # # noqa: C901
     )
 
 
-def run_program(args: list[str], seed: int, timeout: int = 60) -> RunResult:
+def run_program(args: list[str], seed: int, timeout: int = Timeout.TEST) -> RunResult:
```

Comment:
maybe remove default value from here entirely, so we explicitly have to choose a timeout for each run_program?